community/sample-apps/react-static-graphql/README.md

```diff
@@ -0,0 +1,115 @@
+# react-static-graphql
+
+    Boilerplate to get started with react-static site generator, Hasura GraphQL engine and postgres as database using the [react-static](https://github.com/nozzle/react-static) framework.
+
+# Tutorial
+
+- Deploy Postgres and GraphQL Engine on Heroku:
+  
+  [![Deploy to
+  heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+  Please checkout our [docs](https://docs.hasura.io/1.0/graphql/manual/deployment/index.html) for other deployment methods
+
+- Get the Heroku app URL (say `my-app.herokuapp.com`)
+- Create `author` table:
+  
+  Open Hasura console: visit https://my-app.herokuapp.com on a browser  
+  Navigate to `Data` section in the top nav bar and create a table as follows:
+
+  ![Create author table](../gatsby-postgres-graphql/assets/add_table.jpg)
+
+- Insert sample data into `author` table:
+
+  ![Insert data into author table](../gatsby-postgres-graphql/assets/insert_data.jpg)
+
+  Verify if the row is inserted successfully
+
+  ![Insert data into author table](../gatsby-postgres-graphql/assets/browse_rows.jpg)
+
+- Similarly, create an article table with the following data model:
+table: `article`
+columns: `id`, `title`, `content`, `author_id` (foreign key to `author` table's `id`)
```

Comment:
Screenshot for adding FK would be good.

![image](https://cdn.discordapp.com/attachments/408524798839226368/543031052982026260/author_article_fk.png)